api/models/itemDao.js

```diff
@@ -0,0 +1,111 @@
+const appDataSource = require("./dataSource");
+
+
+const seeTeabags = async (data, category) => {
```

Comment:
```suggestion
const getTeabags = async (data, category) => {
```
dao 함수의 naming은 C.R.U.D에 맞춰서 create, get, update, delete 와 리소스의 조합으로 작성되는 것이 보다 일관적이게 작성될 수 있습니다! get + Teabags의 조합으로 작성해주세요~